CMakeLists.txt

```diff
@@ -153,7 +153,7 @@ if(BUILD_WASM)
     endif()
     add_compile_options(-s DISABLE_EXCEPTION_CATCHING=0)
     add_link_options(-sSTACK_SIZE=4MB)
-    if(BUILD_TESTS)
+    if(BUILD_TESTS OR BUILD_EXTENSION_TESTS)
```

Comment:
Should we still differentiate `BUILD_TESTS` and `BUILD_EXTENSION_TESTS`? It looks like they are always used altogether? @acquamarin what do you think?

Reply:
I think there are still some scenarios that we only want to run the main tests but not the extension tests?